### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, open '/home/despairblue/vcs/git/scegratoo3/app/scripts/services/treenodeattribute.js'

### ISSUE THREAD

**Issue by despairblue** [atom/atom#6855]
When changing branches in git while one of the files that are being rewritten (due to the branch switch) is open in atom this happens:

**Atom Version**: 0.199.0
**System**: linux 3.14.39-1-lts
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, open '/home/despairblue/vcs/git/scegratoo3/app/scripts/services/treenodeattribute.js'

```
At events.js:141

Error: ENOENT: no such file or directory, open '/home/despairblue/vcs/git/scegratoo3/app/scripts/services/treenodeattribute.js'
  at Error (native)

```

### Commands

```
     -0:48.2.0 symbols-view:toggle-file-symbols (atom-text-editor.editor.is-focused)
     -0:47.5.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:47.5.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:46.1.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:43 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:40.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:40.9.0 git-plus:new-branch (atom-text-editor.editor)
  6x -0:36.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
  3x -0:27.5.0 editor:move-to-beginning-of-word (atom-text-editor.editor.mini.is-focused)
     -0:25 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "excludeVcsIgnoredPaths": false,
    "disabledPackages": [
      "welcome",
      "metrics",
      "autocomplete",
      "incremental-search",
      "Tern",
      "grunt-runner",
      "tree-view-open-files",
      "tree-view-breadcrumb",
      "atomatigit",
      "themed-settings",
      "linter-scspell",
      "filetype-color",
      "atom-pair",
      "linter-eslint",
      "es-navigation",
      "autocomplete",
      "tab-smart-sort",
      "starred-package-installer",
      "zentabs",
      "linter-jshint",
      "block-cursor",
      "test-status",
      "minimap",
      "minimap-color-highlight",
      "minimap-find-and-replace",
      "minimap-git-diff",
      "minimap-highlight-selected",
      "travis-ci-status",
      "pane-layout-plus",
      "symbols-tree-view",
      "package-cop",
      "navigate"
    ],
    "projectHome": "/home/despairblue/vcs/git/",
    "destroyEmptyPanes": false,
    "themes": [
      "one-dark-ui",
      "solarized-dark-syntax"
    ],
    "autoHideMenuBar": true
  },
  "editor": {
    "fontFamily": "Source Code Pro, Symbola, Droid Sans Fallback",
    "lineHeight": 1.4,
    "showIndentGuide": true,
    "scrollPastEnd": true,
    "invisibles": {},
    "fontSize": 13,
    "softWrap": true
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.1.0
atom-alignment, v0.12.1
atom-color-highlight, v3.0.9
atom-ternjs, v0.5.30
atom-zeal, v0.1.0
autocomplete-emojis, v2.2.2
autocomplete-paths, v1.0.2
block-travel, v1.0.2
bug-report, v0.7.0
color-picker, v1.7.0
compare-files, v0.4.1
docblockr, v0.7.1
editorconfig, v1.0.0
file-icons, v1.5.5
git-log, v0.3.0
git-plus, v4.5.0
go-plus, v3.3.11
grammar-token-limit, v0.1.1
highlight-selected, v0.9.3
indentation-indicator, v0.5.1
javascript-snippets, v1.0.0
keybinding-cheatsheet, v0.0.9
language-dockerfile, v0.1.0
language-dot, v1.6.0
language-fish-shell, v1.0.2
language-haskell, v1.0.0
language-javascript-better, v1.5.0
language-latex, v0.6.1
language-nimrod, v0.1.0
language-puppet, v0.12.0
language-tss, v0.1.0
latex, v0.23.4
linter, v0.12.2
linter-coffeelint, v0.2.3
linter-htmlhint, v0.0.15
linter-js-standard, v0.2.5
linter-js-yaml, v1.0.1
linter-puppet-lint, v0.2.9
linter-shellcheck, v0.0.7
markdown-scroll-sync, v0.3.3
open-last-project, v0.4.1
pdf-view, v0.22.0
project-manager, v1.15.5
regex-railroad-diagram, v0.7.4
tabs-to-spaces, v0.10.0

# Dev
No dev packages
```

**Comment by izuzak** [comment]
Probably one of your packages causing this, but the stack trace doesn't say which one. This should be fixed by atom/notifications#62 which will be included in the next release. 

/cc @kevinsawicki just in case I got this wrong.

**Comment by 50Wliu** [comment]
@izuzak this is actually a duplicate of #6210.

**Comment by izuzak** [comment]
@50Wliu Yeah, maybe. Have you been able to reliably reproduce that other issue? I remember trying a few times, on all OSes, and wasn't able to. Also noticed https://github.com/atom/atom/issues/6846 which was closed by atom/notifications#62, so that made me close this one as well :smile:.

**Comment by despairblue** [comment]
yep, seems to be fixed :+1:
